test/protractor-mac.conf.js

```diff
@@ -0,0 +1,340 @@
+  // if you need to change the browser timeout, use
+  // seleniumArgs: ['-browserTimeout=60']
+  // Ignored if seleniumServerJar is null.
+  seleniumArgs: ['-Dwebdriver.gecko.driver=./node_modules/geckodriver/geckodriver'],
+  // ChromeDriver location is used to help find the chromedriver binary.
+  // This will be passed to the Selenium jar as the system property
+  // webdriver.chrome.driver. If null, Selenium will
+  // attempt to find ChromeDriver using PATH.
+  // chromeDriver: './selenium/chromedriver',
+
+  // ---- 2. To connect to a Selenium Server which is already running ----------
+  // The address of a running Selenium Server. If specified, Protractor will
+  // connect to an already running instance of Selenium. This usually looks like
+  // seleniumAddress: 'http://localhost:4444/wd/hub'
+  seleniumAddress: null,
+
+  // ---- 3. To use remote browsers via Sauce Labs -----------------------------
+  // If sauceUser and sauceKey are specified, seleniumServerJar will be ignored.
+  // The tests will be run remotely using Sauce Labs.
+  sauceUser: null,
+  sauceKey: null,
+  // Use sauceSeleniumAddress if you need to customize the URL Protractor
+  // uses to connect to sauce labs (for example, if you are tunneling selenium
+  // traffic through a sauce connect tunnel). Default is
+  // ondemand.saucelabs.com:80/wd/hub
+  sauceSeleniumAddress: null,
+
+  // ---- 4. To connect directly to Drivers ------------------------------------
+  // Boolean. If true, Protractor will connect directly to the browser Drivers
+  // at the locations specified by chromeDriver and firefoxPath. Only Chrome
+  // and Firefox are supported for direct connect.
+  directConnect: false,
+  // Path to the firefox application binary. If null, will attempt to find
+  // firefox in the default locations.
+  firefoxPath: null,
+
+  // **DEPRECATED**
+  // If true, only ChromeDriver will be started, not a Selenium Server.
+  // This should be replaced with directConnect.
+  chromeOnly: false,
+
+  // ---------------------------------------------------------------------------
+  // ----- What tests to run ---------------------------------------------------
+  // ---------------------------------------------------------------------------
+
+  // Spec patterns are relative to the location of this config.
+  specs: [
+    'integration/**/*.js'
+  ],
+
+  // Patterns to exclude.
+  exclude: ['integration/e2e.js'], // We are temporarily excluding the e2e tests while we transition to the split merge queue
+
+  // Alternatively, suites may be used. When run without a command line
+  // parameter, all suites will run. If run with --suite=smoke or
+  // --suite=smoke,full only the patterns matched by the specified suites will
+  // run.
+  suites: {
+    // smoke: 'spec/smoketests/*.js',
+    rest_api: 'integration/rest_api/*.js' // This suite of tests should only require a running master api, it should not require a node
```

Comment:
i think the suites are updated in master you should compare this to the main protractor.conf in master and make sure this is the same list